charts/rstudio-pm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: rstudio-pm
 description: Official Helm chart for RStudio Package Manager
-version: 0.5.7
+version: 0.5.8
```

Comment:
Do you think we should update this to `0.6.0` since it could be a breaking change for some?

There would be two changes
1. R v3 is no longer installed by default
2. Rely on the default R version inside the container instead of pinning to a specific R version.